workspaces/services/guacamole-azure-dsvm/terraform/main.tf

```diff
@@ -0,0 +1,33 @@
+# Azure Provider source and version being used
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.61.0"
+    }
+  }
+  backend "azurerm" {
+  }
+}
+
+
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_client_config" "current" {}
+
+data "azurerm_resource_group" "ws" {
+  name = "rg-${var.tre_id}-ws-${var.workspace_id}"
+}
+
+data "azurerm_virtual_network" "ws" {
+  name                = "vnet-${var.tre_id}-ws-${var.workspace_id}"
+  resource_group_name = "rg-${var.tre_id}-ws-${var.workspace_id}"
```

Comment:
can this reuse the data element of the rg defined right above instead of redefining the name?

Reply:
done (and in all other services)